railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+ruby <%= "'#{RUBY_VERSION}'" %>
```

Comment:
Sorry @albertoalmagro, totally forgot to answer this. What I meant is not to generate `ruby 2.3.1` without quotes, but to generate `ruby RUBY_VERSION`, which I think would be generated by using `ruby RUBY_VERSION` (wihout `<%=`, `%>` block).